gravitee-apim-console-webui/src/management/integrations/integration-agent/integration-agent.configuration.ts

```diff
@@ -31,3 +31,27 @@ services:
       - gravitee_integration_providers_0_integrationId=\${INTEGRATION_ID}
       - gravitee_integration_providers_0_type=aws-api-gateway
 `;
+
+const solaceConfigurationCode = `
+version: '3.8'
+
+services:
+  integration-agent:
+    image: \${APIM_REGISTRY:-graviteeio}/federation-agent-solace:\${AGENT_VERSION:-latest}
+    restart: always
+    environment:
+      - gravitee_integration_connector_ws_endpoints_0=\${WS_ENDPOINTS}
+      - gravitee_integration_connector_ws_headers_0_name=Authorization
+      - gravitee_integration_connector_ws_headers_0_value=bearer \${WS_AUTH_TOKEN}
+      - gravitee_integration_providers_0_configuration_authToken=\${SOLACE_AUTH_TOKEN}
+      - gravitee_integration_providers_0_configuration_0_appDomains=\${SOLACE_APPLICATION_0_DOMAIN:-}
+      - gravitee_integration_providers_0_configuration_1_appDomains=\${SOLACE_APPLICATION_1_DOMAIN:-}
+      - gravitee_integration_providers_0_integrationId=\${INTEGRATION_ID}
+      - gravitee_integration_providers_0_type=solace
+      - gravitee_integration_providers_0_configuration_url=\${SOLACE_ENDPOINT:-https://apim-production-api.solace.cloud/api/v2/apim}
```

Comment:
I suggest

- little simplification for non helpful without our `.env`
- put at end the optionals configurations

```suggestion
      - gravitee_integration_connector_ws_endpoints_0=\${WS_ENDPOINTS}
      - gravitee_integration_connector_ws_headers_0_name=Authorization
      - gravitee_integration_connector_ws_headers_0_value=bearer \${WS_AUTH_TOKEN}
      - gravitee_integration_providers_0_configuration_authToken=\${SOLACE_AUTH_TOKEN}
      - gravitee_integration_providers_0_integrationId=\${INTEGRATION_ID}
      - gravitee_integration_providers_0_type=solace
      - gravitee_integration_providers_0_configuration_url=https://apim-production-api.solace.cloud/api/v2/apim
      # optionals
      - gravitee_integration_providers_0_configuration_0_appDomains=\${SOLACE_APPLICATION_0_DOMAIN}
      - gravitee_integration_providers_0_configuration_1_appDomains=\${SOLACE_APPLICATION_1_DOMAIN}
```